### ISSUE TITLE
2023-2024 Mobilisation - DEWNR SABG Lichens & Bryophytes

### ISSUE THREAD

**Issue by rosemaryjoconnor** [AtlasOfLivingAustralia/data-management#999]
The State Herbarium of South Australia (SHSA) maintains important bryophyte (>30,000) and lichen
(>6,000) specimens among the more than 1.07 million specimens in the overall collection. These lichen
and bryophyte collections include specimens of national and international importance, including rich
and diverse material of historical and scientific value. 

This grant will enable the provision of dedicated staff time and equipment to image and digitise the
collection’s label information and to make this data accessible and available to the community through
the Atlas of Living Australia (ALA) infrastructure.

**Comment by rosemaryjoconnor** [comment]
**Email: 06/12/2023**

A quick summary of queries:

- Country code handling. (resolved)
- CatalogNumber search/cross-checking for bryophytes (resolved)
- DigiVol expedition - Latitude/Longitude formats
- Lichen dataset, DWC terms, file formats, images

Have suggested a meeting with all 3 involved to define what is needed to progress and how to support them.

**Comment by rosemaryjoconnor** [comment]
**Meeting 12/12/2023**

- Good progress. MS to send through some sample data for issues they are having with lat/lon formats from DigiVol. 
- Discussed images for lichens - will need to discuss with their data team also as dr15865 does not have any images at present and a DWCA is sent to us for manual upload.
- SABG working on a central repository for images
- Will catch up in the new year.

**Comment by rosemaryjoconnor** [comment]
19/12/2023
- [x] Reviewed Bryophyte dataset
- [x] Geocode information only. 
- [x] Code completed updating all formats - mostly UTM formats and special characters needing conversion to DD
- [x] Emailed results to MS. 
- [ ] MS to review and contact in Jan 2024

**Comment by rosemaryjoconnor** [comment]
**01/03/2024**
- [ ] Emailed MS for update

**Comment by rosemaryjoconnor** [comment]
**04/03/2024**
MS responded QA of first datasets in progress, delivery of first dataset not known at this stage.

**Comment by rosemaryjoconnor** [comment]
**08/04/2024**
Email received from MS
_We have finished imaging the bulk of our lichen collection and more than 5,000 records have been transcribed on DigiVol.
We have largely completed the quality assurance work on the bryophyte data (c. 15,000 records) and are almost ready to deliver the first c. 2000 lichen specimens to the ALA._

**Comment by rosemaryjoconnor** [comment]
**07/05/2024**
MS emailed to say the first round of the data has been successfully pushed to ALA with more to come. 
Have requested a record count for mobilisation.

**Comment by rosemaryjoconnor** [comment]
**03/06/2024**
Meeting with MS and Juliet.

- Data on the way to be complete. 
- ALA scheduled data loads on 20th of each month. So 20/06/2024 will be a load and the final load will be on 20/07/2024

**Comment by rosemaryjoconnor** [comment]
**24/07/2024**

Emailed MS requesting confirmation of load and record count

**Comment by rosemaryjoconnor** [comment]
**14/10/2024**

Dataset with approx 23K records provided - this is not final. 

A few things:

- Field/column names were not Darwin Core Terms 
- Column 2 had no header at all - I set it to institutionCode as it was the field with only value being 'DAF'
- Institution code needs to be: 'QDPC'
- Trailing '-' characters in dates that a YYYY-MM only 
- The 'from' and 'To' visit dates have been combined in to a single 'eventDate' field separated by a '/' 

After combining the two tabs from the spreadsheets there were about 62 duplicate records - I've attached those for you. 

Emailed JB with all information. Provided files with:

- DWC column mappings
- duplicates
- ALA metadata template

`dr22342` successfully loaded in databox: https://collections-test.ala.org.au/public/show/co52
